lib/bill_forward/entities/payment_method.rb

```diff
@@ -1,27 +1,27 @@
 module BillForward
-	class PaymentMethod < MutableEntity
-  		@resource_path = BillForward::ResourcePath.new("payment-methods", "paymentMethod")
+  class PaymentMethod < MutableEntity
```

Comment:
I rather prefer tabs (or 4 spaces) also. Intellij uses double spaces by default with ruby. Doing a (VERY) quick search it looks like it's ruby's suggested standard.